khelo/lib/ui/flow/tournament/detail/tournament_detail_screen.dart

```diff
@@ -263,7 +263,7 @@ class _TournamentDetailScreenState
     if (state.tournament == null) {
       return;
     }
-    if (state.tournament!.teams.length >= state.tournament!.type.minTeamReq) {
+    if (state.tournament!.teams.length == state.tournament!.type.minTeamReq) {
```

Comment:
let's revert this change, we will allow match scheduling for any number of team **BUT** greater than or equal to minTeamRequired.